LeetCodeTests/LongestCommonSubsequenceTests.cs

```diff
@@ -0,0 +1,50 @@
+namespace Tests;
+using LeetCodeSolutions;
+public class LongestCommonSubsequenceTests
+{
+  private LongestCommonSubsequenceSolution _solution;
+  public LongestCommonSubsequenceTests()
+  {
+    _solution = new();
+  }
+  [Fact]
+  public void TextLengthsOfOne1()
+  {
+    Assert.Equal(1, _solution.LongestCommonSubsequence("a", "a"));
```

Comment:
create a test function and parameters object to bring this in line with general code style

spaces between tests, too